sql/catalyst/pom.xml

```diff
@@ -104,6 +104,10 @@
       <groupId>org.antlr</groupId>
       <artifactId>antlr4-runtime</artifactId>
     </dependency>
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+    </dependency>
```

Comment:
Can we move this to the other module? `sql/catalyst` looks a little surprising to me.

Reply:
I think `sql/catalyst` used to depend on this which came from Hadoop side, and since we now moved to shaded version, we'll need to explicitly add this dependency so that compilation won't break.

Reply:
Thanks!